client/src/pages/mainPage/components/footer/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import "./Footer.css"
+import{BsGithub} from "react-icons/bs"
+import { BsLinkedin } from "react-icons/bs";
+
+export default function Footer() {
+  return (
+		<footer>
```

Comment:
Perhaps format this? In VSCode I think you can run `Shift + Alt + F`

Reply:
Sure